src/utils/tests/scopes.js

```diff
@@ -103,4 +106,251 @@ describe("scopes", () => {
       });
     });
   });
+
+  describe("getVisibleVariablesFromScope", function() {
+    /* These are real-life pauseInfo and frames with some shadowed variables
+     * coming from the following JS code:
+     *
+     * (function() {
+     *   var a = 'a';
+     *   var b = 'b';
+     *   var c = 'c';
+     *
+     *   function func(b, d) {
+     *     var c = 'cc';
+     *     debugger; // This is where we are paused.
+     *   }
+     *
+     *   document.querySelector('button').onclick =
+     *     () => func.call('this', 'bb', 'dd');
+     *  })();
+     *
+     *
+     * We use a IIFE because we don't support global variables yet, so this is
+     * necessary to get variables in the outer scope. This can be removed once
+     * getVisibleVariablesFromScope supports global variables.
+     *
+     * `b` is shadowed by a function parameter, `c` is shadowed by a local
+     * variable, but `a` is not shadowed. `this` is a string containing te value
+     * `"this"`.
+     */
+
+    const frame = {
```

Comment:
This is not so simple but I found that it's better to have something closer to the reality.